apps/beeai-cli/src/beeai_cli/__init__.py

```diff
@@ -26,6 +27,9 @@
 app.add_typer(beeai_cli.commands.mcp.app, name="mcp", no_args_is_help=True, help="Manage MCP servers and toolkits.")
 app.add_typer(beeai_cli.commands.build.app, name="", no_args_is_help=True, help="Build agent images.")
 app.add_typer(beeai_cli.commands.auth.app, name="", no_args_is_help=True, help="Beeai login.")
+app.add_typer(
```

Comment:
Having the login logic split between `auth` and `resource_server` command modules is messy. Ideally we should make this a single CLI namespace, like `beeai server login`. Also, it seems that in code, the names "resource" and "resource server" and "server" are used interchangeably, and since all seem to refer to the same thing, we should unify on something clear like `server` or `host`.

Reply:
Alternatively we could call it `context` like Docker